datahub-web-react/src/app/AppProviders.tsx

```diff
@@ -13,7 +14,9 @@ export default function AppProviders({ children }: Props) {
         <AppConfigProvider>
             <UserContextProvider>
                 <EducationStepsProvider>
-                    <QuickFiltersProvider>{children}</QuickFiltersProvider>
+                    <QuickFiltersProvider>
+                        <SearchContextProvider>{children}</SearchContextProvider>
```

Comment:
So happy we have this now.

Reply:
This is probably where our subscription state could go in the future if we ever wanted to hoist it up.